rmgpy/data/base.py

```diff
@@ -1363,6 +1386,9 @@ def load_entry(self, label, group=None, molecule=None, species=None, shortDesc='
             item=item,
             short_desc=shortDesc,
             long_desc=longDesc.strip(),
+            metal=metal,
+            facet=facet,
+            site=site,
         )
 
     def save_entry(self, f, entry, name='entry'):
```

Comment:
Why don't you need to edit this to save the metal, facet and site attributes? Won't it delete the attribute value if someone saves the database?

Reply:
Thanks for catching this. @mazeau please can you take a look and fix or explain?

Reply:
Alright, I think I've fixed this! Thanks for catching it